src/main/java/application/controllers/MainController.java

```diff
@@ -440,6 +440,7 @@ public void toggleShowReferenceStrain() {
      */
     public void strainSelection(ArrayList<String> ref, List<String> s) {
         graphController.getGraph().reset();
+        graphController.getZoomBox().reset();
```

Comment:
Please check if this is correct? Because I removed it with a reason, but I might be mistaking